src/test/java/com/rtb/projectmanagementtool/task/TaskServletTest.java

```diff
@@ -0,0 +1,86 @@
+package com.rtb.projectmanagementtool.task;
+
+import static org.mockito.Mockito.*;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.*;
+import com.google.appengine.api.datastore.dev.LocalDatastoreService;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import java.io.*;
+import javax.servlet.ServletException;
+import javax.servlet.http.*;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.*;
+
+public class TaskServletTest extends Mockito {
+
+  private TaskServlet servlet;
+  private HttpServletRequest request;
+  private HttpServletResponse response;
+
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(
+          new LocalDatastoreServiceTestConfig()
+              .setAutoIdAllocationPolicy(LocalDatastoreService.AutoIdAllocationPolicy.SEQUENTIAL));
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+    servlet = new TaskServlet();
+    request = mock(HttpServletRequest.class);
+    response = mock(HttpServletResponse.class);
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  @Test
+  public void testDoGet() throws IOException, ServletException {
```

Comment:
This test looks good.  Just a couple suggestions for the next PR:
-  If we're expecting JSON results, you can maybe parse the JSON into an object.  It doesn't have to be a TaskData object, but maybe a gson object would suffice.  This will allow us to assert the whole object, which would compare the results field by field.  Also if the order of the fields in the json string changes, asserting the object itself would allow for this
- If you'd rather keep the assert with a json string, look into formatting as a "raw string literal" or a "multi-line string literal" to improve readability, [eg](https://dzone.com/articles/jdk-12-raw-string-literals)
- Look into mocking the data for the test.  Either if you can wrap the datastore service with mockito (I'm not sure if this is possible), or by passing a local datastore into your servlet like how we did for the controller.  This allows for 2 things:  1.  you can control the output of the datastore and 2.  it wouldn't affect the production data.